### PR TITLE
added readme file of button component and update map readme file

### PR DESCRIPTION
**Pull request by rohitgupta274509**
Readme link:-
[https://github.com/typerefinery-ai/typerefinery-websight/blob/feature/add-readme-file-for-button-component/application/backend/src/main/resources/apps/typerefinery/components/forms/fields/button/README.md](url)

please review and approve it.

### CODE REVIEW

**Review by Rahul139-stack — APPROVED** [review]

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/README.md [hunk 1]
```diff
@@ -156,16 +154,46 @@ These fields are available for input by the authors. These fields are used in te
...
+            <td>Text Alignment</td>
+            <td>Default</td>
+            <td>Contains alignment of the text.</td>
+        </tr>
+        <tr>
+            <td rowspan="4"> Style</td>
+            <td>Class name</td>
+            <td>None</td>
+            <td>Add custom style classes.</td>
+        </tr>
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/widgets/map/README.md:182]: image

Why map component is updated with these styles ?

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/README.md [hunk 2]
```diff
@@ -156,16 +154,46 @@ These fields are available for input by the authors. These fields are used in te
...
-            <td>Id</td>
-            <td></td>
-            <td>Controls the css and styling.</td>
+            <td>Width - M breakpoint</td>
+            <td>12 Col</td>
+            <td>Contains medium grid width.</td>
+        </tr>
+        <tr>
+            <td>Width - L breakpoint</td>
+            <td>3 Col</td>
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/widgets/map/README.md:169]: By default map takes 12 in L Breaking point.

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/README.md (hunk 1 above)
Comment (rohitgupta274509) [reply application/backend/src/main/resources/apps/typerefinery/components/widgets/map/README.md:188]: now its changed.

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
image

Can you please update the text of the L, M, and S breakpoints to something like this?

L - Large Screen Break Points will be applicable to screens larger than 992 px.

Refer to the below picture for exact dimensions:

image

**Comment by Harikarthyk** [comment]
FYI: https://getbootstrap.com/docs/5.0/layout/breakpoints/

**Comment by rohitgupta274509** [comment]
>
Updated readme file as request changes.

**Comment by rohitgupta274509** [comment]
> image
> 
> I could not see icon Position attr in the button component.

>Updated Icon position attribute.

**Review by Harikarthyk — APPROVED** [review]

**Pull request merged**